src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,400 @@
+
+**Reading:**
+
+- [Setting up Webots](/guides/tools/webots-setup)
+- [Webots User Guide](https://cyberbotics.com/doc/guide/index)
+
+**References:**
+
+- [NUWebots](https://github.com/NUbots/NUWebots) [GitHub]
+- [Webots (Original Version)](https://github.com/cyberbotics/webots) [GitHub]
+- [Webots (TC Version)](https://github.com/RoboCup-Humanoid-TC/webots) [GitHub]
+
+</details>
+
+<details>
+<summary> Mathematics </summary>
+
+Mathematics is an integral part of any robotics system.
+
+**Reading:**
+
+- [Mathematics](/system/foundations/mathematics#linear-algebra) [NUbook]
+- [Eigen C++ Linear Algebra Library](http://eigen.tuxfamily.org/dox/)
+- [Wolfram Mathworld](http://mathworld.wolfram.com/)
+
+**Videos:**
+
+- [3Blue1Brown: Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab)
+- [MIT Open Courseware: Linear Algebra](https://ocw.mit.edu/courses/mathematics/18-06-linear-algebra-spring-2010/video-lectures/)
+
+**References:**
+
+- [Wolfram Alpha](https://www.wolframalpha.com/)
+- [Eigen Source](https://gitlab.com/libeigen/eigen)
+
+</details>
+
+<details>
+<summary> Python </summary>
+
+Python is used to create tools in the main robot codebase.
+
+**Videos:**
+
+- [Python Essential Training](https://www.linkedin.com/learning/python-essential-training-2/) [LinkedIn Learning]
+
+</details>
+
+<details>
+<summary> Machine Learning </summary>
+
+Primarily used in the vision system, but could be used in any system.
+
+**Videos:**
+
+- [3Blue1Brown: Neural Networks](https://www.youtube.com/playlist?list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi) [YouTube]
+- [DeepLizard: Machine Learning & Deep Learning Fundamentals](https://www.youtube.com/watch?v=gZmobeGL0Yg&list=PLZbbT5o_s2xq7LwI2y8_QtvuXZedL6tQU) [YouTube]
+
+**References:**
+
+- [Official TensorFlow Documentation](https://www.tensorflow.org/overview)
+
+</details>
+
+<details>
+<summary> Log and Play Back Data </summary>
+
+**Reading:**
+
+- [Data Recording and Playback Guide](/guides/main/data-recording-playback) [NUbook]
+- [Logging](/system/subsystems/logging) [NUbook]
+
+</details>
+
+<details>
+<summary> Lab Induction </summary>
+
+The lab induction must be completed before getting swipe card access to the lab.
+
+**Reading:**
+
+- [Lab Induction](/team/induction) [NUbook]
+
+**Tutorials:**
+
+- NUbots lab inductions are run for new team members after they join.
+
+</details>
+
+## Behaviour
+
+<details>
+<summary>General</summary>
+
+**Reading:**
+
+- [Behaviour](/system/subsystems/behaviour) [NUbook]
+
+**References:**
+
+- [GameController](https://github.com/RoboCup-Humanoid-TC/GameController) [GitHub]
+- [Communication Protocol](https://github.com/RoboCup-Humanoid-TC/RobocupProtocol) [GitHub]
+
+</details>
+
+## DevOps and Quality Assurance
+
+<details>
+<summary> The Build System </summary>
+
+**Reading:**
+
+- [Build System](/system/foundations/build-system) [NUbook]
```

Comment:
We should probably link the CI page as well.

Reply:
See Docker comment.